python/cuvs_bench/pyproject.toml

```
@@ -33,9 +37,6 @@ classifiers = [
 [project.urls]
 Homepage = "https://github.com/rapidsai/raft"
```

This homepage should be updated.
```suggestion
Homepage = "https://github.com/rapidsai/cuvs"
```